lib/modules/manager/github-actions/extract.ts

```diff
@@ -84,8 +87,10 @@ function extractWithRegex(content: string): PackageDependency[] {
         versioning: dockerVersioning.id,
         depType: 'action',
         replaceString,
-        autoReplaceStringTemplate: `${quotes}{{depName}}${path}@{{#if newDigest}}{{newDigest}}${quotes}{{#if newValue}} # {{newValue}}{{/if}}{{/if}}{{#unless newDigest}}{{newValue}}${quotes}{{/unless}}`,
-        ...customRegistryUrlsPackageDependency,
+        autoReplaceStringTemplate: `${quotes}${registryUrl ?? ''}{{depName}}${path}@{{#if newDigest}}{{newDigest}}${quotes}{{#if newValue}} # {{newValue}}{{/if}}{{/if}}{{#unless newDigest}}{{newValue}}${quotes}{{/unless}}`,
```

Comment:
I find this hard to read, is not possible to extract this?

Reply:
```suggestion
        autoReplaceStringTemplate: `${quotes}${registryUrl}{{depName}}${path}@{{#if newDigest}}{{newDigest}}${quotes}{{#if newValue}} # {{newValue}}{{/if}}{{/if}}{{#unless newDigest}}{{newValue}}${quotes}{{/unless}}`,
```

I'm not sure how, most if this is constant. only `quotes`, `registryUrl` and `path` are js variables here, everything else are handlebar templates 🤔